src/server/plugins/react-render/index.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+const React = require('react')
+const ReactDOM = require('react-dom/server')
+const ReactRouter = require('react-router')
+const Helmet = require('react-helmet')
+const Boom = require('boom')
+const filter = require('lodash.filter')
+const forEach = require('lodash.foreach')
+
+function register (server, options = {}, next) {
+  const routes = options.routes
+  const assets = server.plugins.huyang.stats.assets
+  console.log(assets)
+  // Extract scripts to pass to served html
+  const bundles = filter(assets, (asset) => asset.chunkNames.indexOf('main') !== -1)
```

Comment:
No need for the double `!!`

`const bundles = filter(assets, (asset) => ~asset.chunkNames.indexOf('main'))`